src/libs/ReportActionsUtils.ts

```diff
@@ -372,7 +373,7 @@ function shouldReportActionBeVisible(reportAction: OnyxEntry<ReportAction>, key:
         return false;
     }
 
-    if (isPendingRemove(reportAction)) {
+    if (isPendingRemove(reportAction) && !isThreadParentMessage(reportAction)) {
```

Comment:
```suggestion
    if (isPendingRemove(reportAction) && reportAction.childVisibleActionCount === 0) {
```
How would you feel about keeping `isThreadParentMessage` as is? The more I think about it, I think it's not quite appropriate to use that function at all since it's not really what we're looking for - we're only looking for, effectively, `isParentOfVisibleThread` or something like that. Right?

I think this is the only real comparison we need, since only chats can be pending remove (I think)?

Let me know if that makes sense or if you have another suggestion.